llvm/docs/CMake.rst

```diff
@@ -836,6 +836,12 @@ enabled sub-projects. Nearly all of these variable names begin with
   The full list, as of March 2023, is:
   ``AArch64;AMDGPU;ARM;AVR;BPF;Hexagon;Lanai;LoongArch;Mips;MSP430;NVPTX;PowerPC;RISCV;Sparc;SystemZ;VE;WebAssembly;X86;XCore``
 
+  You can also specify ``host`` or ``Native`` to automatically detect and
```

Comment:
Just document the common `host` and omit `Native`?

Reply:
It is good to only _promote_ one canonical symbolic name, e.g. `host`. However, for someone reading the documentation for understanding some project found on the net, it is really important that _all_ accepted names are documented.

Reply:
Okay, thanks.